File: Models/Competition/LeagueModel.cs
namespace champi.Models.Competition
{
    public class LeagueModel
    {
        public long Id { get; set; }
        public long CompetitionStepId { get; set; }
        public int TeamCount { get; set; }
        public bool IsHomeAway { get; set; }
        public int PeerToPeerPlayCount { get; set; }
        public int RiseTeamCount { get; set; }
        public int FallTeamCount { get; set; }
    }
}

File: ClientApp/src/app/models/competition/league.model.ts
import { LeagueTeamModel } from './league-team.model';

export class LeagueModel {
    id: number;
    competitionStepId: number;
    teamCount: number;
    isHomeAway: boolean;
    peerToPeerPlayCount: number;
    riseTeamCount: number;
    fallTeamCount: number;
    wonPoint: number;
    drawPoint: number;
    lostPoint: number;
    leagueTeams: LeagueTeamModel[];
}
